src/vcModals.cpp

```diff
@@ -656,10 +656,89 @@ void vcModals_DrawProjectInfo(vcState *pProgramState)
+          {
+            size_t startImagePos = 0;
+            size_t openBracketPos = 0;
+            size_t closeBracketPos = 0;
+            size_t openParanthesisPos = 0;
+            size_t closeParanthesisPos = 0;
+
+            udStrchr(pInfo + firstIndex, "!", &startImagePos);
+            startImagePos += firstIndex;
+            udStrchr(pInfo + startImagePos, "[", &openBracketPos);
+            openBracketPos += startImagePos;
+            udStrchr(pInfo + openBracketPos, "]", &closeBracketPos);
+            closeBracketPos += openBracketPos;
+            udStrchr(pInfo + closeBracketPos, "(", &openParanthesisPos);
+            openParanthesisPos += closeBracketPos;
+            udStrchr(pInfo + openParanthesisPos, ")", &closeParanthesisPos);
+            closeParanthesisPos += openParanthesisPos;
+            
+            const int64_t invalidLen = udStrlen(pInfo);
+
+            if (startImagePos == invalidLen ||
+              openBracketPos == invalidLen ||
+              closeBracketPos == invalidLen ||
+              openParanthesisPos == invalidLen ||
+              closeParanthesisPos == invalidLen)
+              break;
+
+            size_t lastCharPos = closeParanthesisPos;
+            if (openBracketPos == startImagePos + 1 &&
+              closeBracketPos > openBracketPos &&
+              openParanthesisPos == closeBracketPos + 1 &&
+              closeParanthesisPos > openParanthesisPos)
+            {
+              const char *pSubStr = udStrndup(pInfo + startStrIndex, startImagePos - startStrIndex);
+              pProgramState->projectInfoTextures.infoStrings.PushBack(pSubStr);
+
+              const char *pTexPath = udStrndup(pInfo + (openParanthesisPos + 1), closeParanthesisPos - openParanthesisPos - 1);
+              uint32_t w, h;
+              vcTexture *pNewTexture = nullptr;
+              vcTexture_CreateFromFilename(&pNewTexture, pTexPath, &w, &h);
+              pProgramState->projectInfoTextures.textures.PushBack(pNewTexture);
+              pProgramState->projectInfoTextures.textureSizes.PushBack(ImVec2((float)w, (float)h));
+
+              firstIndex = (int)(lastCharPos + 1);
+              startStrIndex = firstIndex;
+            }
+            else
+            {
+              firstIndex = (int)(lastCharPos + 1);
+            }
+          }
+          
+          const char *pSubStr = udStrndup(pInfo + firstIndex, udStrlen(pInfo) - firstIndex);
+          pProgramState->projectInfoTextures.infoStrings.PushBack(pSubStr);
+        }
+
+        for (size_t i = 0; i < pProgramState->projectInfoTextures.infoStrings.length; ++i)
+        {
+          ImGui::TextWrapped("%s", pProgramState->projectInfoTextures.infoStrings[i]);
+          if (i < pProgramState->projectInfoTextures.infoStrings.length - 1 && pProgramState->projectInfoTextures.textures[i] != nullptr)
+            ImGui::Image(pProgramState->projectInfoTextures.textures[i], pProgramState->projectInfoTextures.textureSizes[i]);
```

Comment:
The task explicitly states that if the image isn't available to render the alt text.